server/lib/genome/normalizers/interaction_claim_type.rb

```diff
@@ -71,42 +71,52 @@ def self.name_normalizer(val)
+        when 'positive modulator', 'positive allosteric modulator', 'regulator (upregulator)', 'enhancer',
+            'modulator (allosteric modulator)'
           'positive modulator'
-        when 'inhibitor','inhibitor, competitive', 'gating inhibitor', 'inhibitor; antagonist; blocker', 'inhibitor (gating inhibitor)', 'growth_inhibition', 'inhibition', 'weak inhibitor', 'aggregation inhibitor', 'inhibition of synthesis', "translocation inhibitor", 'inhibits downstream inflammation cascades', 'inactivator', 'inihibitor', 'inhibitors', 'anti-angiogenic.', 'allosteric inhibitor','antagonist','antisense inhibitor'
+        when 'inhibitor', 'inhibitor, competitive', 'gating inhibitor', 'inhibitor; antagonist; blocker',
+            'inhibitor (gating inhibitor)', 'growth_inhibition', 'inhibition', 'weak inhibitor',
+            'aggregation inhibitor', 'inhibition of synthesis', 'translocation inhibitor',
+            'inhibits downstream inflammation cascades', 'inactivator', 'inihibitor', 'inhibitors',
+            'anti-angiogenic.', 'allosteric inhibitor', 'antagonist', 'antisense inhibitor'
           'inhibitor'
-        when 'blocker','channel blocker', 'blocker (channel blocker)', 'nucleotide exchange blocker'
+        when 'blocker', 'channel blocker', 'blocker (channel blocker)', 'nucleotide exchange blocker'
           'blocker'
         when 'antisense', 'sirna drug'
           'antisense oligonucleotide'
-        when 'binding agent','binding', 'binder (minor groove binder)', 'breaker'
+        when 'binding agent', 'binding', 'binder (minor groove binder)', 'breaker'
           'binder'
-        when 'negative modulator','negative allosteric modulator','disrupting agent','incorporation into and destabilization', 'intercalation', 'desensitize the target', 'disrupter', 'intercalator', 'downregulator','allosteric antagonist'
+        when 'negative modulator', 'negative allosteric modulator', 'disrupting agent',
+            'incorporation into and destabilization', 'intercalation', 'desensitize the target', 'disrupter',
+            'intercalator', 'downregulator', 'allosteric antagonist'
           'negative modulator'
-        when 'inhibitory immune response', 'car-t-cell-therapy(dual specific)', 'immunomodulator', 'immunomodulator (immunostimulant)', 'immune response agent', 'car-t-cell-therapy', 'immune response agent', 'immunostimulant', 'immunostimulator', 'Radioimmunotherapy'
+        when 'inhibitory immune response', 'car-t-cell-therapy(dual specific)', 'immunomodulator',
+            'immunomodulator (immunostimulant)', 'immune response agent', 'car-t-cell-therapy',
+            'immunostimulant', 'immunostimulator', 'Radioimmunotherapy'
           'immunotherapy'
         when 'component of'
           'product of'
-        when 'opener'
+        when 'opener'  # TODO: duplicate w/ 'other/unknown' -- how to resolve?
```

Comment:
'opener' is included with the 'other/unknown' group above, so this can't be reached -- we should remove it from one or the other groupings